frontend/src/pages/projects/utils.ts

```diff
@@ -32,3 +32,9 @@ export const getNotebookDescription = (notebook: NotebookKind): string =>
 
 export const getPvcDisplayName = (pvc: PersistentVolumeClaimKind): string =>
   getDisplayNameFromK8sResource(pvc);
+
+export const getPvcDescription = (pvc: PersistentVolumeClaimKind): string =>
+  getDescriptionFromK8sResource(pvc);
+
+export const getPvcTotalSize = (pvc: PersistentVolumeClaimKind): string =>
```

Comment:
Update my code to use this too -- maybe this is already shared when we share render components... but we should definitely both do the same logic.